content/events/ftc-at-ethcc.md

```diff
@@ -1,11 +1,18 @@
 ---
 eventName: FTC at EthCC
+eventDescription: "Funding the Commons is focused on new models of sustainable public goods funding and value alignment in open source networks. We bridge the public goods community across Web2, Web3, research, philanthropy and industry. We do this by convening builders, academics, and funders in both traditional conference and "open space" formats."
```

Comment:
@laeccles Please update the content in the description's last line - if open space is required to be enclosed within double quotations, use the escape character \ before the double quotations just like below - 
\"open space\"

Reply:
I have updated this